backend/src/test/java/com/github/redstylzz/backend/service/PaymentServiceTest.java

```diff
@@ -20,19 +23,24 @@
 
     private final IPaymentRepository paymentRepo = mock(IPaymentRepository.class);
     private final ICategoryRepository categoryRepository = mock(ICategoryRepository.class);
-    private final PaymentService underTest = new PaymentService(paymentRepo, categoryRepository);
+    private final CategoryService categoryService = mock(CategoryService.class);
+    private final PaymentService underTest = new PaymentService(paymentRepo, categoryRepository, categoryService);
 
+    @BeforeAll
+    static void init() {
+        TestDataProvider.mockUUID();
```

Comment:
this method annotation is not clear in its intention, from naming I would assume it returns a value, which is not put into a variable. If this method returns a value it should be saved to a variable which can be used later or state a clear intention, e.g. TestDataProvider.clearDatabase();